### PR TITLE
fastFVA added as optional

### PR DESCRIPTION
**Pull request by Gpreciat**
The sampling function chrrSampler.m has as default function fluxVariability.m to perform FVA. fastFVA it is optional.

Adapted in the tutorial.
**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: src/analysis/FVA/fluxVariability.m [hunk 1]
```diff
@@ -133,12 +133,7 @@
 LPproblem.c = model.c;
 LPproblem.lb = model.lb;
 LPproblem.ub = model.ub;
-if ~isfield(model,'csense')
-    LPproblem.csense(1:nMets) = 'E';
-else
-    LPproblem.csense = model.csense;
-end
-
+LPproblem.csense(1:nMets) = 'E';
```
Comment (laurentheirendt) [inline src/analysis/FVA/fluxVariability.m]: These lines have been added in PR #843 (rebase error). May you add them again? Thanks.

File: src/analysis/FVA/fluxVariability.m [hunk 2]
```diff
@@ -156,11 +151,7 @@
 LPproblem.S = LPproblem.A;%needed for sparse optimisation
 
 %solve to generate initial basis
-if ~isfield(model,'osense')
-    LPproblem.osense = -1;
-else
-    LPproblem.osense = model.osense; 
-end
+LPproblem.osense = -1;
```
Comment (laurentheirendt) [inline src/analysis/FVA/fluxVariability.m]: These lines have been added in PR #843 (rebase error). May you add them again? Thanks.

File: tutorials/analysis/uniformSampling/tutorial_uniformSampling.m [hunk 3]
```diff
@@ -99,20 +91,24 @@
...
-% with$<math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mrow><msup><mrow><mtext>?
+% with$<math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mrow><msup><mrow><mtext> 
 % </mtext></mrow><mrow><mn>1</mn></mrow></msup></mrow></math>$.
 % 
 % We start by loading the model with its flux bounds and the objective function 
 % (ATP demand reaction). We set the maximum glucose uptake rate to 18.5 mmol/gDW/hr. 
 % To explore the entire space of feasible steady state fluxes we also remove the 
 % cellular objective.
 
+options.useFastFVA = 0; % to use fluxVariability
```
Comment (laurentheirendt) [inline tutorials/analysis/uniformSampling/tutorial_uniformSampling.m]: Wouldn't this be `options.useFastFVA = false`?

File: src/analysis/FVA/fluxVariability.m (hunk 1 above)
Comment (Gpreciat) [inline src/analysis/FVA/fluxVariability.m]: sure

File: src/analysis/FVA/fluxVariability.m (hunk 2 above)
Comment (Gpreciat) [inline src/analysis/FVA/fluxVariability.m]: sure

File: tutorials/analysis/uniformSampling/tutorial_uniformSampling.m (hunk 3 above)
Comment (Gpreciat) [inline tutorials/analysis/uniformSampling/tutorial_uniformSampling.m]: true

**Comment by syarra** [comment]
Thanks @Gpreciat and @laurentheirendt!

**Pull request merged**